llama-index-integrations/readers/llama-index-readers-notion/llama_index/readers/notion/base.py

```diff
@@ -190,40 +190,49 @@ def search(self, query: str) -> List[str]:
         return page_ids
 
     def load_data(
-        self, page_ids: List[str] = [], database_ids: Optional[List[str]] = None
+        self, 
+        page_ids: List[str] = [], 
```

Comment:
Nit: we shouldn't be using an empty list as a default kwarg

Reply:
yeah i'm not sure how this decision was made, might leave it for now